tests/trace/test_base_object_classes.py

```diff
@@ -53,15 +53,15 @@ def test_pythonic_creation(client: WeaveClient):
     nested_obj = base_objects.TestOnlyNestedBaseObject(b=3)
     top_obj = base_objects.TestOnlyExample(
         primitive=1,
-        nested_base_model=TestOnlyNestedBaseModel(a=2),
+        nested_base_model=TestOnlyNestedBaseModel(a=2, aliased_property_alias=3),
         nested_base_object=weave.publish(nested_obj).uri(),
     )
     ref = weave.publish(top_obj)
 
     top_obj_gotten = weave.ref(ref.uri()).get()
 
     assert isinstance(top_obj_gotten, base_objects.TestOnlyExample)
-    assert top_obj_gotten.model_dump() == top_obj.model_dump()
+    assert top_obj_gotten.model_dump(by_alias=True) == top_obj.model_dump(by_alias=True)
```

Comment:
We could consider this trick to avoid having to `by_alias` so much:
https://github.com/pydantic/pydantic/discussions/8725#discussioncomment-11722435